includes/Dashboard/Templates/Orders.php

```diff
@@ -125,6 +125,9 @@ function handle_order_export() {
             dokan_order_csv_export( $user_orders );
             exit();
         }
+        
+        // Allow dev to create custom CSV
+        do_action('dokan/order/custom_exports', $post_data);
```

Comment:
change the hook name with: `dokan_after_handle_order_export`

Reply:
name changed